keyboards/polykybd/.claude/skills/keycap-layout-preview/keycap_preview.py

```
@@ -0,0 +1,247 @@
+
+
+def sheet(cells, path, scale=5, gap=26, pad=22):
+    """cells: [(ink_set, label)] laid out in one row. Writes a PNG."""
+    font = ImageFont.truetype(
+        '/usr/share/fonts/truetype/dejavu/DejaVuSansMono-Bold.ttf', 15)
+    caps = [_cap(ink, scale) for ink, _ in cells]
+    cw, chh = caps[0].size
+    out = Image.new('RGB', (pad * 2 + cw * len(caps) + gap * (len(caps) - 1),
+                            pad * 2 + chh + 26), _BG)
+    d = ImageDraw.Draw(out)
+    for i, (cap, (_, label)) in enumerate(zip(caps, cells)):
```

Ruff (0.16.1)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)